src/components/layout/sider.tsx

```diff
@@ -0,0 +1,402 @@
+import React, { CSSProperties, useState } from 'react'
+import {
+  CanAccess, ITreeMenu, useIsExistAuthentication, useLogout, useTitle, useTranslate, useRouterContext,
+  useRouterType, useLink, useMenu, useRefineContext, useActiveAuthProvider, pickNotDeprecated, useWarnAboutChange
+} from '@refinedev/core'
+import { useThemedLayoutContext } from '@refinedev/mui'
+import { ThemedTitleV2 as DefaultTitle } from './title'
+import ChevronLeft from '@mui/icons-material/ChevronLeft'
+import Dashboard from '@mui/icons-material/Dashboard'
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+import ListOutlined from '@mui/icons-material/ListOutlined'
+import Logout from '@mui/icons-material/Logout'
+import Box from '@mui/material/Box'
+import Collapse from '@mui/material/Collapse'
+import Drawer from '@mui/material/Drawer'
+import IconButton from '@mui/material/IconButton'
+import List from '@mui/material/List'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import Paper from '@mui/material/Paper'
+import Tooltip from '@mui/material/Tooltip'
+import type { RefineThemedLayoutV2SiderProps } from '@refinedev/mui'
+
+export const ThemedSiderV2: React.FC<RefineThemedLayoutV2SiderProps> = ({ Title: TitleFromProps, render, meta, activeItemDisabled = false }) => {
+  const { siderCollapsed, setSiderCollapsed, mobileSiderOpen, setMobileSiderOpen } = useThemedLayoutContext()
+
+  const drawerWidth = () => {
+    if (siderCollapsed) return 56
+    return 240
+  }
```

Comment:
Not an issue, just an optional suggestion
```suggestion
  const drawerWidth = () => siderCollapsed ? 56 : 240
```